pennylane/transforms/qcut/__init__.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This subpackage contains quantum function transforms for cutting quantum circuits.
+"""
+import pennylane as qml
+
+from .qcut import (
+    MeasureNode,
+    PrepareNode,
+    replace_wire_cut_node,
+    replace_wire_cut_nodes,
+    tape_to_graph,
+    fragment_graph,
+    graph_to_tape,
+    expand_fragment_tape,
+    CutStrategy,
+    cut_circuit,
+    cut_circuit_mc,
+    expand_fragment_tapes_mc,
+    qcut_processing_fn_sample,
+    qcut_processing_fn_mc,
+    qnode_execution_wrapper_mc,
+    contract_tensors,
+    qcut_processing_fn,
+    qnode_execution_wrapper,
+    kahypar_cut,
+    place_wire_cuts,
+    find_and_place_cuts,
+    _remove_existing_cuts,
+    _graph_to_hmetis,
+    _qcut_expand_fn,
+    _process_tensor,
+    _to_tensors,
+    _reshape_results,
+    _get_measurements,
```

Comment:
I believe not for PennyLane. It sounded like we were worried about breaking changes for people using qcut, so I included them in case someone is accessing them via `transforms.qcut._whatever_private_method`.

Of course that doesn't matter as long as the `__gettattr__` is included here. But for next PR (I'll tag you on it in a minute), I've imported everything, so that everything is accessible the way it was when qcut was a file in transforms. Should I skip the private methods?

Reply:
@trbromley  Do we care about the import paths of private methods?